### DISCUSSION TITLE
v0.44.0

### DISCUSSION THREAD

**Discussion by WillTheDeveloper** [WillTheDeveloper/StudyPortal#133] [Announcements]
**Full Changelog**: https://github.com/WillTheDeveloper/StudyPortal/compare/v0.43.1...v0.44.0

<hr /><em>This discussion was created from the release <a href='https://github.com/WillTheDeveloper/StudyPortal/releases/tag/v0.44.0'>v0.44.0</a>.</em>

**Comment by hrszpuk** [comment]
Nice

**Reply by WillTheDeveloper** [reply]
Nice

**Reply by hrszpuk** [reply]
Will the pre-release have most of the features with the v1.0.0 being some final polish or are there going to be major changes between the two versions? @WillTheDeveloper

**Reply by WillTheDeveloper** [reply]
The pre-release which is what it is in its current state has most of the features that have been designed and implemented and the v1 release will be a polished version for both mobile and desktop users since at the moment it is primarily desktop optimized. @hrszpuk

**Reply by hrszpuk** [reply]
What's your mobile solution going to be? Does Laravel support you porting the web app to mobile? @WillTheDeveloper

**Reply by WillTheDeveloper** [reply]
At the moment, the mobile solution will be students using the platform in a browser. I would like to create an app one day but I have not yet found a reliable android framework or language that will allow for an easy implementation of study portal. There is a way to port the laravel app directly to a mobile app but this is just a react native app I believe. My main concern at the moment though is getting the web version off the ground, into V1, have some students on it and then go from there. If that makese sense? @hrszpuk

**Reply by hrszpuk** [reply]
I think react native is a fine choice for this kind of application but I understand you want to research more into different languages and frameworks before porting. Good luck with v1.0.0 release! 😄

**Reply by WillTheDeveloper** [reply]
Furthermore, one of the prerequistites for a mobile version of the platform would first be to have a completed API backend where data and authentication will be able to take place of the back of the main website/server itself.

**Reply by WillTheDeveloper** [reply]
> I think react native is a fine choice for this kind of application but I understand you likely want to research more into different languages and frameworks before porting. Good luck with v1.0.0 release! 😄

Much appreciated :) And yeah, I want to make sure that we pick a framework and language that will be sustainable and easy to implement into what we currently have. Only time will tell :D
